services/core-web/src/components/mine/Incidents/MineIncident.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { useEffect, useState } from "react";
+import React, { FunctionComponent, Requireable, useEffect, useState } from "react";
 import { bindActionCreators } from "redux";
 import { flattenObject } from "@common/utils/helpers";
 import { connect } from "react-redux";
 import { Link, withRouter, useParams, useLocation, matchPath } from "react-router-dom";
 import { change, submit, getFormSyncErrors, getFormValues, touch, isDirty } from "redux-form";
 import { Tag } from "antd";
 import { ArrowLeftOutlined, EnvironmentOutlined } from "@ant-design/icons";
-import PropTypes from "prop-types";
+import PropTypes, { string } from "prop-types";
```

Comment:
Please check if PropTypes is used anywhere here.

Reply:
Removed unused imports